File: languages/java/app-encryption/src/test/java/com/godaddy/asherah/appencryption/testhelpers/ByteArray.java
package com.godaddy.asherah.appencryption.testhelpers;

public class ByteArray {
  public static Boolean isAllZeros(byte[] input) {
    for (byte b : input) {
      if (b != 0) {
        return false;
      }
    }
    return true;
  }
}

File: languages/csharp/AppEncryption/AppEncryption.Tests/AppEncryption/TestHelpers/ByteArray.cs
using System.Linq;

namespace GoDaddy.Asherah.AppEncryption.Tests.AppEncryption.TestHelpers
{
    public static class ByteArray
    {
        public static bool IsAllZeros(byte[] input)
        {
            return input.All(b => b == 0);
        }
    }
}
